tests/compare_diractories_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# this code will check if there is a new module that dont have a test diractory, in that case it will bring back
+# the missing test file that the user will need to add to the repo.
+
+dir1="tests"
+dir2="modules"
+
+dir1_dirs=$(find "$dir1" -type d -maxdepth 1 -mindepth 1 -exec basename {} \;)
```

Comment:
4. If the directory is empty the code will fail in the terraform init part, i can modify the code so that it will check that the directory is not empty.